### PR TITLE
Add org.gtk.Gtk3theme.Akwa-light

### PR DESCRIPTION
**Pull request by berkiyo**

### CODE REVIEW

**Comment by berkiyo** [comment]
bot, build org.gtk.Gtk3theme.Akwa-light

**Comment by berkiyo** [comment]
bot, build org.gtk.Gtk3theme.Akwa-light

File: org.gtk.Gtk3theme.Akwa-light.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "id": "org.gtk.Gtk3theme.Akwa-light",
+    "branch": "3.22",
+    "runtime": "org.freedesktop.Platform",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "19.08",
```
Comment (barthalion) [inline org.gtk.Gtk3theme.Akwa-light.json:7]:
```suggestion
    "runtime-version": "20.08",
```

**Comment by barthalion** [comment]
bot, build org.gtk.Gtk3theme.Akwa-light

**Pull request closed without merging**